### PR TITLE
Updating DFKI documentation

### PR DESCRIPTION
**Pull request by VPoser**

### CODE REVIEW

File: workflow_creation/utilities_dfki.md [hunk 1]
```diff
@@ -1,2 +1,276 @@
...
+<input type="button" value="Send 'finishedExecution'" onclick="cont_wf()" />
+</div></body></html>
+
+
+```
+
+### Inputs
+| Parameter name | Description |
+--- | :--- |
+| `session token` | Session token to be used throughout the workflow |
```
Comment (rojsc) [inline workflow_creation/utilities_dfki.md]: Wrong parameter names

File: workflow_creation/advanced_workflow_nesting.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
 # Advanced workflow creation: Nesting workflows
-to be written
\ No newline at end of file
+
+Workflows can also be used as services within other workflows, which can ease the creation of more sophisticated workflows, as thereby complex workflows can be divided into different tool-chains or sub-workflow.
```
Comment (rojsc) [inline workflow_creation/advanced_workflow_nesting.md]: Typo: "sub-workflow" -> "sub-workflows"

File: workflow_creation/advanced_branching_looping.md [hunk 3]
```diff
@@ -1,2 +1,109 @@
...
\ No newline at end of file
+# Advanced workflow creation: Branching and looping
+
+To create more sophisticated workflows, the workflow editor offers branching and loops:
+
+Need to perform a case distinction and execute different services based on the result? Make use of the [branching feature](#branching).
+
+Want to execute a service multiple times? There is no need to manually call it multiple times, make use of the [loops](#loops).
+
+
```
Comment (rojsc) [inline workflow_creation/advanced_branching_looping.md]: Suggestion: Add short note that these advanced topics have to be implemented using the xml notation, and say how to get that notation visible.

File: workflow_creation/advanced_branching_looping.md [hunk 4]
```diff
@@ -1,2 +1,109 @@
...
+Perform a simple case distinction and execute different services depending on the result.
+
+### Steps:
+
+![wfe_1.png](img_workflows/wfe_1.png)
+
+- In the "Workflow Editor"-UI below the "Service URI"-field click on "append if tags" to add the basic syntax to your workflow
+
+```
```
Comment (rojsc) [inline workflow_creation/advanced_branching_looping.md]: Add `xml` after the three single quotes to enable syntax highlighting

File: workflow_creation/advanced_branching_looping.md [hunk 5]
```diff
@@ -1,2 +1,109 @@
...
+
+Both features support a simple condition scheme, denoted in xml, which enables integer comparisons:
+
+```
+<condition>COMMAND(VARIABLE, VALUE)</condition>
+```
+
+The COMMAND may one of the following: `greaterThan`, `greaterOrEqual`, `equalTo`, `lessThan`, `lessOrEqual`.
+The condition will compare the value of the VARIABLE with VALUE based on the COMMAND.
+
```
Comment (rojsc) [inline workflow_creation/advanced_branching_looping.md]: Suggestion: Add a note that any of the workflow and service parameters currently "active" can be used as variables.

Question: Is that correct? ;-)

File: workflow_creation/advanced_branching_looping.md [hunk 6]
```diff
@@ -1,2 +1,109 @@
...
+If you need to execute a service multiple time, you can make use of the loop feature:
+
+### Steps:
+
+![wfe_1.png](img_workflows/wfe_1.png)
+
+- In the "Workflow Editor"-UI below the "Service URI"-field click on "append loop tags" to add the basic syntax to your workflow.
+
+```
```
Comment (rojsc) [inline workflow_creation/advanced_branching_looping.md]: Add syntax highlighting

**Comment by rojsc** [comment]
One more thing: What about the "Append split tags" button in WFE? What does that do? Should we describe it somewhere here as well?

**Comment by VPoser** [comment]
Regarding the "Append split tags" button: It is supposed to enable parallel computation/service execution in workflows. So far I did not find the time to test this feature extensively, so I have not yet provided a tutorial to praise this feature.

**Pull request merged**